gammapy/modeling/models/core.py

```diff
@@ -428,6 +430,18 @@ def read(cls, filename):
     def from_yaml(cls, yaml_str, path=""):
         """Create from YAML string."""
         data = yaml.safe_load(yaml_str)
+        if "metadata" in data.keys():
+            creator_version = data["metadata"]["creator"].split()[1]
```

Comment:
If version is important, I would suggest to split it  directly in the `CreatorMetaData`:
maybe having `creation.software` and `creation.version` would be simpler